README.md

```diff
@@ -11,4 +11,4 @@ eCardnomics is a **desktop flashcard application to quickly create, manage, and
 Useful links:
 * [User Guide](https://ay2021s1-cs2113-t14-2.github.io/tp/UserGuide.html)
 * [Developer Guide](https://ay2021s1-cs2113-t14-2.github.io/tp/DeveloperGuide.html)
-* [About Us](AboutUs.md)
+* [About Us](https://ay2021s1-cs2113-t14-2.github.io/tp/AboutUs.html)
```

Comment:
Is it possible to just link to `docs/AboutUs.md`?

Reply:
Actually, I am not sure but I am just following the same method as the User Guide and Developer Guide since those links work on the web page.

Reply:
Alright then let's go with your method.